src/lib/globalIntervals.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
This file should be moved to a `bin` directory within the `src` directory since it's not a function that should be directly accessed by the end user.